notebooks/TFSegFormer.ipynb

```diff
@@ -0,0 +1,322 @@
+{
```

Comment:
<p>We can do a <code>print(logits.shape)</code> and add a comment on the shape of the logits: <code>(batch_size, num_labels, height, width)</code>. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-diver/segformer-tf-transformers/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>